README.md

```diff
@@ -20,30 +20,23 @@ import App from ./App.vue
 const vueLifecycles = singleSpaVue({
   Vue,
   appOptions: {
-    el: '#mount-location',
     render: h => h(App),
   }
 });
 
-export const bootstrap = [
-  vueLifecycles.bootstrap,
-];
+export const bootstrap = vueLifecycles.bootstrap;
```

Comment:
nitpick: can this show an example of using functions as the lifecycle hooks, as sometimes I will use bootstrap as a function in order to pass the set public path method.

Reply:
Using an array for lifecycles is explained here, and will always be possible with any single-spa application: https://single-spa.js.org/docs/building-applications.html#registered-application-lifecycle.

The reason I changed it in the Readme is that many people don't implement their own lifecycles and just copy paste from the Readme and find the array to be a bit more confusing.

Reply:
ok thats cool :)